packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.js

```diff
@@ -2848,7 +2848,14 @@ describe('ReactSuspenseWithNoopRenderer', () => {
       foo.setState({suspend: false});
     });
 
-    expect(Scheduler).toHaveYielded(['Foo']);
+    expect(Scheduler).toHaveYielded([
+      // First setState
+      'Foo',
+      // Second setState. This update was scheduled while we were in the
+      // middle of rendering the previous update, so it was pushed to a separate
+      // batch to avoid invalidating the work-in-progress tree.
+      'Foo',
+    ]);
```

Comment:
The comment seems incorrect. The first value was actually yielded by the second setState since the first setState scheduled an update that would suspend and never be pinged. In addition, if there's another act block before this act block, the assertion here would fail again(as there would be only one yielded value):
```
await ReactNoop.act(async () => {
  foo.setState({ a: 1 });
});
expect(Scheduler).toHaveYielded(['Foo']);
```
The expectation is no longer obvious due to how we determine **the equal or higher priority lanes** now.